go.mod

```diff
@@ -5,12 +5,15 @@ go 1.12
 require (
 	github.com/AccelByte/bloom v0.0.0-20180915202807-98c052463922
 	github.com/AccelByte/go-jose v2.1.4+incompatible
+	github.com/AccelByte/go-restful-plugins/v3 v3.1.2
 	github.com/cenkalti/backoff v2.2.1+incompatible
+	github.com/codahale/hdrhistogram v0.0.0-20161010025455-3a0bb77429bd // indirect
 	github.com/gorhill/cronexpr v0.0.0-20180427100037-88b0669f7d75
+	github.com/opentracing/opentracing-go v1.1.0
 	github.com/patrickmn/go-cache v2.1.0+incompatible
 	github.com/pkg/errors v0.8.1
-	github.com/spaolacci/murmur3 v0.0.0-20180118202830-f09979ecbc72 // indirect
-	github.com/stretchr/testify v1.3.0
-	github.com/willf/bitset v1.1.10 // indirect
-	golang.org/x/crypto v0.0.0-20190308221718-c2843e01d9a2 // indirect
+	github.com/sirupsen/logrus v1.4.1
+	github.com/stretchr/testify v1.4.0
 )
+
+replace github.com/AccelByte/go-restful-plugins/v3 v3.1.2 => ./tmp/go-restful-plugins
```

Comment:
Don't forget to change this and remove tmp folder after https://github.com/AccelByte/go-restful-plugins merged

Reply:
@leowind yes